tests/std/tests/P0448R4_spanstream/test.cpp

```diff
@@ -0,0 +1,1014 @@
+        assert(static_cast<test_buf*>(first.rdbuf())->egptr() == end(buffer1));
+        assert(static_cast<test_buf*>(first.rdbuf())->pbase() == nullptr);
+        assert(static_cast<test_buf*>(first.rdbuf())->pptr() == nullptr);
+        assert(static_cast<test_buf*>(first.rdbuf())->epptr() == nullptr);
+        assert(static_cast<test_buf*>(second.rdbuf())->eback() == buffer2);
+        assert(static_cast<test_buf*>(second.rdbuf())->gptr() == buffer2);
+        assert(static_cast<test_buf*>(second.rdbuf())->egptr() == end(buffer2));
+        assert(static_cast<test_buf*>(second.rdbuf())->pbase() == buffer2);
+        assert(static_cast<test_buf*>(second.rdbuf())->pptr() == buffer2);
+        assert(static_cast<test_buf*>(second.rdbuf())->epptr() == end(buffer2));
+
+        first.swap(second);
+        assert(first.span().data() == buffer2);
+        assert(second.span().data() == buffer1);
+        assert(static_cast<test_buf*>(second.rdbuf())->eback() == buffer1);
+        assert(static_cast<test_buf*>(second.rdbuf())->gptr() == buffer1);
+        assert(static_cast<test_buf*>(second.rdbuf())->egptr() == end(buffer1));
+        assert(static_cast<test_buf*>(second.rdbuf())->pbase() == nullptr);
+        assert(static_cast<test_buf*>(second.rdbuf())->pptr() == nullptr);
+        assert(static_cast<test_buf*>(second.rdbuf())->epptr() == nullptr);
+        assert(static_cast<test_buf*>(first.rdbuf())->eback() == buffer2);
+        assert(static_cast<test_buf*>(first.rdbuf())->gptr() == buffer2);
+        assert(static_cast<test_buf*>(first.rdbuf())->egptr() == end(buffer2));
+        assert(static_cast<test_buf*>(first.rdbuf())->pbase() == buffer2);
+        assert(static_cast<test_buf*>(first.rdbuf())->pptr() == buffer2);
+        assert(static_cast<test_buf*>(first.rdbuf())->epptr() == end(buffer2));
+
+        swap(first, second);
+        assert(first.span().data() == buffer1);
+        assert(second.span().data() == buffer2);
+        assert(static_cast<test_buf*>(first.rdbuf())->eback() == buffer1);
+        assert(static_cast<test_buf*>(first.rdbuf())->gptr() == buffer1);
+        assert(static_cast<test_buf*>(first.rdbuf())->egptr() == end(buffer1));
+        assert(static_cast<test_buf*>(first.rdbuf())->pbase() == nullptr);
+        assert(static_cast<test_buf*>(first.rdbuf())->pptr() == nullptr);
+        assert(static_cast<test_buf*>(first.rdbuf())->epptr() == nullptr);
+        assert(static_cast<test_buf*>(second.rdbuf())->eback() == buffer2);
+        assert(static_cast<test_buf*>(second.rdbuf())->gptr() == buffer2);
+        assert(static_cast<test_buf*>(second.rdbuf())->egptr() == end(buffer2));
+        assert(static_cast<test_buf*>(second.rdbuf())->pbase() == buffer2);
+        assert(static_cast<test_buf*>(second.rdbuf())->pptr() == buffer2);
+        assert(static_cast<test_buf*>(second.rdbuf())->epptr() == end(buffer2));
+    }
+
+    {
+        // rdbuf
+        // ^^^ look above ^^^
+    }

```

Comment:
Are these tests work in progress? This comment is very confusing.

Reply:
I wanted to say that I feel like I have sufficiently tested the `rdbuf` interface in the tests above

Reply:
I think it should just be removed.

Reply:
Would you be happy with a simple comment `// rdbuf already tested above`

Reply:
👍